BlueMapCommon/webapp/public/map_data_provider.php

```diff
@@ -2,19 +2,18 @@
 
 // !!! SET YOUR SQL-CONNECTION SETTINGS HERE: !!!
 
+$driver   = 'pgsql'; //PDO driver name, check the PDO documentation for available drivers at https://www.php.net/manual/en/pdo.drivers.php. Common ones are 'pgsql' or 'mysql'.
 $hostname = '127.0.0.1';
-$port     = 3306;
-$username = 'root';
+$port     = 5432; // Remember to change this value to match your database configuration
+$username = '';
 $password = '';
 $database = 'bluemap';
```

Comment:
I'd like to keep the default at the mysql settings that we had before, since (afaik) more people are using mysql than postgresql :)

Reply:
Seems like a good idea